Dockerfile

```diff
@@ -57,7 +57,6 @@ RUN echo "Starting the build ..." &&\
 ############################################################
 
 # Volumes
-VOLUME ["/etc/ssl/certs"]
```

Comment:
This looks wrong to me, this is still necessary as it contains the output `ca-certificate.crt` and `cacerts`.